tools/FxCopRules/DataWebRules.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Is this change intentional?

Reply:
No it is not. Will revert.

Reply:
@ysanghi, out of curiosity, which version of Visual Studio are you loading the projects with? If VS2017, then there is a known issue with building this particular project and I recommend just unloading it from your solutions if it's bothering you. I have an open PR that addresses this issue but it hasn't been checked in yet.

Reply:
Will do it next time. For now will revert this change. As long as there are no other comments will make this commit.